packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```diff
@@ -0,0 +1,245 @@
+    jsonPath: string;
+    name: string | null;
+    secondaryInformation: string | null;
+};
+
+type Dependent = {
+    rootGraphqlObjectType: string;
+    rootId: string;
+    rootColumnName: string;
+    jsonPath: string;
+    name: string | null;
+    secondaryInformation: string | null;
+};
+
+interface Query {
+    item: {
+        id: string;
+        dependencies?: { totalCount: number; nodes: Array<Dependency> };
+        dependents?: { totalCount: number; nodes: Array<Dependent> };
+    };
+}
+
+type QueryVariables = {
+    offset: number;
+    limit: number;
+    forceRefresh?: boolean;
+};
+
+interface DependencyListProps {
+    query: TypedDocumentNode<Query, QueryVariables>;
+    variables: Record<string, unknown>;
+}
+
+const pageSize = 10;
+
+export const DependencyList = ({ query, variables }: DependencyListProps) => {
+    const intl = useIntl();
+    const entityDependencyMap = useDependenciesConfig();
+    const contentScope = useContentScope();
+    const apolloClient = useApolloClient();
+    const history = useHistory();
+
+    const dataGridProps = useDataGridRemote({ queryParamsPrefix: "dependencies", pageSize: pageSize });
+
+    const { data, loading, error, refetch } = useQuery<Query, QueryVariables>(query, {
+        variables: {
+            offset: dataGridProps.page * dataGridProps.pageSize,
+            limit: dataGridProps.pageSize,
+            ...variables,
+        },
+    });
+
+    const columns: GridColDef<DependencyItem>[] = [
+        {
+            field: "nameInfo",
+            headerName: intl.formatMessage({ id: "comet.dependencies.dataGrid.nameAndInfo", defaultMessage: "Name/Info" }),
+            sortable: false,
+            flex: 1,
+            renderCell: ({ row }) => {
+                return (
+                    <sc.NameInfoWrapper>
+                        <sc.NameInfoTypography color="text.primary">{row.name}</sc.NameInfoTypography>
+                        <sc.NameInfoTypography color="text.secondary">{row.secondaryInformation}</sc.NameInfoTypography>
+                    </sc.NameInfoWrapper>
+                );
+            },
+        },
+        {
+            field: "type",
+            headerName: intl.formatMessage({ id: "comet.dependencies.dataGrid.type", defaultMessage: "Type" }),
+            sortable: false,
+            renderCell: ({ row }) => <sc.StyledChip label={entityDependencyMap[row.graphqlObjectType]?.displayName ?? row.graphqlObjectType} />,
+        },
+        {
+            field: "actions",
+            headerName: "",
+            sortable: false,
+            renderCell: ({ row }) => {
+                const dependencyObject = entityDependencyMap[row.graphqlObjectType] as DependencyInterface | undefined;
+
+                if (dependencyObject === undefined) {
+                    console.warn(
+                        `Cannot log URL because no implementation of DependencyInterface for ${row.graphqlObjectType} was provided via the DependenciesConfig`,
```

Comment:
```suggestion
                        `Cannot load URL because no implementation of DependencyInterface for ${row.graphqlObjectType} was provided via the DependenciesConfig`,
```

This should only be logged during development IMO